Dockerfile

```diff
@@ -6,19 +6,19 @@ RUN apt-get update && apt-get install --yes python3-pip libsodium-dev
 # Python dependencies
 ENV LANG C.UTF-8
 RUN pip3 install --upgrade pip
-RUN pip3 install gunicorn
-
-# Set git commit ID
-ARG COMMIT_ID
-ENV COMMIT_ID=${COMMIT_ID}
-RUN test -n "${COMMIT_ID}"
+RUN pip3 install talisker pipenv
```

Comment:
I think it would be cleaner/more efficient to remove the Talisker dependency in this file and install it from the requirements file.

Reply:
Yes. I think I did actually add it to requirements. So I don't think it's actually needed here, nor is pipenv any more.

Reply:
Fixed I believe